### ISSUE TITLE
Documentation and tutorials

### ISSUE THREAD

**Issue by VladNov** [NateWr/content-layout-control#14]
Hi Nate,

I had the same idea but to do it with widgets, then i found you plugin. Awesome work with components, this feature is really needed in WP instead of all the heavy and bloated builders. This much cleaner and easier approach for content creation.

Do you have some sort of docs about how to work with CLC? How to create new components and add more fields like selects, Iris color picker, etc.?

**Comment by NateWr** [comment]
HI @VladNov,

Unfortunately, I don't really have any docs in place. You can see how to integrate it into a theme with [this demo theme](https://github.com/NateWr/clc-demo-theme). And if you want to see how custom components are created, take a look at [this plugin](https://github.com/NateWr/totc-layout-control) that I wrote. It adds a number of components that I use in my own themes.

**Comment by VladNov** [comment]
Thank you for your reply.

Do i need to add the contents to "github-updater-lite" directory? Once activated it returned two errors about non existing file or directory.

**Comment by NateWr** [comment]
You can download the latest [release package](https://github.com/NateWr/totc-layout-control/releases) for TOTC Layout Control. If you want to check out the repository, you'll need to initialize the submodules:

```
git submodule update --init --recursive
```

**Comment by VladNov** [comment]
Thank you.

I've installed the plugin, but now i do not see the Content Layout Control panel in the Customizer. Do i need add something into the theme to make it appear?

Sorry about all these questions...

**Comment by NateWr** [comment]
Yeah, here's an example from my base theme:

https://github.com/NateWr/totcbase/blob/master/functions.php#L36-L50

**Comment by VladNov** [comment]
Working great! 

Really appreciate all your help.

One last question, do you have an example of color field? I mean the default WP Iris picker.

**Comment by NateWr** [comment]
No, I generally use it for content construction rather than theme-wide styles.

**Comment by VladNov** [comment]
I thought to add a background color option to section.

You should suggest this library to Automattic to make it part of the core. Thank you very much for all your help and your time, really appreciate it.
